### PR TITLE
udpate installation docs for OpenYurt

### PR DESCRIPTION
**Pull request by rambohe-ch**

### CODE REVIEW

**Comment by luc99hen** [comment]
The duplicate docs in the version7.0 folder is not covered.

File: docs/installation/manually-setup.md [hunk 1]
```diff
@@ -96,15 +129,9 @@ scp -i <yourt-ssh-identity-file> /tmp/yurthub-ack.yaml root@us-west-1.192.168.0.
...
-
-Please refer to this [document](https://github.com/openyurtio/openyurt.io/blob/master/docs/installation/yurt-tunnel-tutorial.md#5-setup-the-yurt-tunnel-manually) to setup Yurttunnel manually.
+#### 4.1.2 Configure Kubelet
 
-## Reset the Kubelet
-
-By now, we have setup all required components for the OpenYurt cluster, next, we only need to reset the
-kubelet service to let it access the apiserver through the yurthub (The following steps assume that we are logged
-in to the edge node as the root user).
+we need to reset the kubelet service to let it access the apiserver through the yurthub (The following steps assume that we are logged in to the edge node as the root user).
```
[the remark this answers is not in the document]
Comment (rambohe-ch) [reply docs/installation/manually-setup.md:134]: fixed

File: docs/installation/manually-setup.md [hunk 2]
```diff
@@ -96,15 +129,9 @@ scp -i <yourt-ssh-identity-file> /tmp/yurthub-ack.yaml root@us-west-1.192.168.0.
...
 As kubelet will connect to the Yurthub through http, so we create a new kubeconfig file for the kubelet service.
```
[the remark this answers is not in the document]
Comment (rambohe-ch) [reply docs/installation/manually-setup.md:135]: fixed

File: docs/installation/manually-setup.md [hunk 3]
```diff
@@ -126,15 +153,41 @@ preferences: {}
 EOF
 ```
 
-In order to let kubelet to use the new kubeconfig, we edit the drop-in file of the kubelet
-service (i.e., `/etc/systemd/system/kubelet.service.d/10-kubeadm.conf`)
+In order to let kubelet to use the new kubeconfig, we edit the drop-in file of the kubelet service (i.e., `/etc/systemd/system/kubelet.service.d/10-kubeadm.conf`)
```
[the remark this answers is not in the document]
Comment (rambohe-ch) [reply docs/installation/manually-setup.md:156]: fixed

File: docs/installation/manually-setup.md [hunk 4]
```diff
@@ -63,29 +72,53 @@ Then edit the config/setup/yurt-controller-manager.yaml
...
+```bash
+$ kubectl apply -f config/setup/yurt-tunnel-server.yaml
+```
+
+And, apply the yurt-tunnel-agent yaml:
+```bash
+kubectl apply -f config/setup/yurt-tunnel-agent.yaml
+```
+
+After the agent and the server are running, we should check yurt-tunnel can work by executing command like `kubectl logs/exec`
```
[the remark this answers is not in the document]
Comment (rambohe-ch) [reply docs/installation/manually-setup.md:100]: fixed

File: docs/installation/manually-setup.md [hunk 5]
```diff
@@ -2,24 +2,31 @@
...
 $ kubectl get nodes
 NAME                     STATUS   ROLES    AGE     VERSION
-us-west-1.192.168.0.87   Ready    <none>   3d23h   v1.14.8-aliyun.1
-us-west-1.192.168.0.88   Ready    <none>   3d23h   v1.14.8-aliyun.1
+us-west-1.192.168.0.87   Ready    <none>   3d23h   v1.20.11
+us-west-1.192.168.0.88   Ready    <none>   3d23h   v1.20.11
 ```
-and we will use node `us-west-1.192.168.0.87` as the cloud node.
+
+OpenYurt should change kubernetes component configurations to adapt to edge environment. Please complete the following configurations for `Kube-Controller-Manager`, `CoreDNS`, `KubeProxy`。
```
[the remark this answers is not in the document]
Comment (rambohe-ch) [reply docs/installation/manually-setup.md:19]: fixed

File: docs/installation/openyurt-prepare.md [hunk 6]
```diff
@@ -1,17 +1,28 @@
...
 ---
-## 1.Background
+## 0.Background
 
-OpenYurt should change kubernetes component configurations to adapt to edge environment. The components include: CoreDNS,KubeProxy etc。
+OpenYurt should change kubernetes component configurations to adapt to edge environment. The components include: Kube-Controller-Manager, CoreDNS,KubeProxy etc。
+
+## 1. Kube-Controller-Manager Adjustment
+
+In order to make the yurt-controller-mamanger to work properly, we need to turn off the default nodelifecycle controller in Kube-Controller-Manager.
```
[the remark this answers is not in the document]
Comment (rambohe-ch) [reply docs/installation/openyurt-prepare.md:10]: fixed

**Comment by rambohe-ch** [comment]
@luc99hen all comments have been fixed, please take a look.

**Pull request merged**